lib/at_ex/USSD.ex

```diff
@@ -0,0 +1,36 @@
+defmodule AtEx.USSD do
+  @moduledoc """
+  This module handles building valid response for the Africas Talking USSD endpoint
+  """
+
```

Comment:
It will be a good idea to transform the request from Africa is talking and create a USSD defstruct.
 defstruct sessionId: nil, phoneNumber: nil, networkCode: nil, serviceCode: nil, text: nil

It will increase code readability and usability.

Reply:
@johninvictus sounds doable but it will be the onus of the developer to consume the post request and produce a list of responses, that gives us a clear separation of concerns